docs/user-guide/ze-working-with-jobs.md

```diff
@@ -17,6 +17,21 @@
 
    ![Download Spool](../images/ze/ZE-jobs-download-spool.gif)
 
+## Sorting jobs
+
+1. Expand **JOBS** in the **Side Bar**.
+2. Click on the **Sort** icon to the right of a profile.
+3. Select the **Sort Direction** option and select either **Ascending** or **Descending** from the **picker** field.
+4. Select a sort type from the list of available options:
+   - **Job ID (default)**
+   - **Name**
```

Comment:
Name seems like a duplicate since Job Name occurs later in this list

Reply:
fixed -- thank you!